src/Raven.Server/Documents/Handlers/AI/Agents/ConversationHandler.cs

```diff
@@ -714,7 +671,7 @@ private async Task<bool> TryHandleActionResponses(JsonOperationContext context)
             await HandleSubAgentCalls(context, subAgentsActions);
         }
 
-        if (_document.OpenActionCalls.Count > 0)
+        if (_document.OpenActionCalls.Any(x => x.Value.Type != AiAgentActionRequestType.SubAgent) || _childUserCalls.Any())
```

Comment:
can we have here a single open action of the type `SubAgent` without any child user calls?

Reply:
also `_childUserCalls.Count > 0`